shellstub.py

```
@@ -0,0 +1,90 @@
+from __future__ import annotations
+import json
+import os
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)